src/components/stepWizard/example usage/page.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactNode } from "react";
+
+import Content from "./content";
+import Header from "./header";
+import Footer from "./footer";
+import Step from "../Step";
+
+const Step1: ReactNode = (
+  <Step stepNumber={1}>
+    <Header />
+    <Content />
+    <Footer stepNumber={1} />
+  </Step>
+);
+
+const Step2: ReactNode = (
+  <Step stepNumber={2}>
+    <Header />
+    <Content />
+    <Footer stepNumber={2} />
+  </Step>
+);
+
+const Step3: ReactNode = (

```

Comment:
I would try to move all "example" code to the StepWizard story file and delete the "example usage" folder and everything in it, that way we aren't pushing these examples to main. This may become easier once we have real components and you don't have to use fake ones. I think at the time of writing this comment, there are some real ones that exist on main now that you can probably pull in.
This will also probably fix the linting errors - because I'm thinking the linter doesn't like the fact that components are being created (and not exported) at the top-level of the file.